server/node_filter.go

```diff
@@ -783,3 +803,41 @@ func nodeIsOutdated(n *cke.Node, current *corev1.Node, taintCP bool) bool {
 
 	return false
 }
+
+// SSHNotConnectedNodes returns nodes which are not connected via SSH from targets
```

Comment:
> SSHNotConnectedNodes returns nodes which are not connected via SSH from targets

SSHNotConnectedNodes returns nodes that are not connected via SSH _out of_ targets.